### PR TITLE
OSDOCS-8392: Migrate "Deploy an Application - Storage" from rosaworkshop.io

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8392](https://issues.redhat.com//browse/OSDOCS-8392): Migrate "Deploy an Application - Storage" from rosaworkshop.io

Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OSDOCS-8392

Link to docs preview:
https://75376--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH - Can I please get QE review of this PR? Thanks!

**Comment by 0kashi** [comment]
Looks good.  There seems to be an extra period at the end of "Crashing the pod." which is a section heading.

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.adoc [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-04-30
+
+== Persistent volumes
+ROSA supports storing persistent volumes with either link:https://aws.amazon.com/ebs/[Amazon Web Services (AWS) Elastic Block Store (EBS)] or link:https://aws.amazon.com/efs/[AWS Elastic File System (EFS)]. For more information about persistent volume storage, see xref:../../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[Understanding persistent storage].
+
+== Using persistent volumes
+Use the following procedures to create a file, store it on a persistent volume in your cluster, and confirm that it still exits after pod failure and re-creation.
+
+=== Viewing a persistent volume claim
```
Comment (duanwei33) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.adoc:18]: Before "Viewing a persistent volume claim", I think it's better to introduce how to create the PVC and how the PVC is consumed by the application/pod, or add the step on how the OSToy app and its PVC come.

**Comment by tsze-redhat** [comment]
https://75376--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.html#viewing-a-persistent-volume-claim
says 
Navigate to the [OpenShift Cluster Manager](https://console.redhat.com/openshift).

There is no Storage link from OpenShift Cluster Manager.

**Comment by jneczypor** [comment]
> https://75376--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.html#viewing-a-persistent-volume-claim says Navigate to the [OpenShift Cluster Manager](https://console.redhat.com/openshift).
> 
> There is no Storage link from OpenShift Cluster Manager.
> 
> You probably are referring to the OpenShift console of the cluster - not OCM.

Hi @tsze-redhat - Thanks so much! Is there an official name for the "OpenShift console of the cluster"? I haven't been able to find one. Thanks!

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.adoc (hunk 1 above)
Comment (jneczypor) [reply cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-storage.adoc:23]: I added some additional resources for users that want to learn more before beginning the procedure. Thanks!

**Comment by tsze-redhat** [comment]
I think it is called Web Console
https://docs.openshift.com/container-platform/4.15/web_console/web-console.html

Storage's doc also just refers to it as console
https://docs.openshift.com/container-platform/4.15/storage/persistent_storage/persistent-storage-aws.html#creating-the-persistent-volume-claim

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**